modules/glib/2.82.2.bcr.2/overlay/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
```

Comment:
For overlay files and patches that are identical to the version in `2.82.2.bcr.1`, can you use a symlink to reference the previous version?

```
$ cd modules/glib/2.82.2.bcr.2/overlay
$ diff -u ../../2.82.2.bcr.1/overlay/BUILD.bazel BUILD.bazel
$ ln -sf ../../2.82.2.bcr.1/overlay/BUILD.bazel BUILD.bazel
```

This makes the PR easier to review.

Reply:
Done.